src/parkapi_sources/converters/freiburg_disabled_static/models.py

```diff
@@ -56,6 +56,7 @@ def to_static_parking_spot_input(self) -> StaticParkingSpotInput:
             static_data_updated_at=datetime.now(tz=timezone.utc),
             lat=round_7d(Decimal(point.y)),
             lon=round_7d(Decimal(point.x)),
+            type=ParkingSpotType.ON_STREET,
```

Comment:
This is also not valid for several datasets: I do see P+R Parkplatz, Parkplatz Reiterstrasse or Parkplatz Ecke Hildastr right away, and have several other datasets which might have the same issue. Still do that?